reflex-algos/src/test/scala/com/parallelmachines/reflex/test/reflexpipeline/DagTestUtil.scala

```diff
@@ -4,7 +4,7 @@ import java.nio.file.{Files, Path, Paths}
 
 import com.parallelmachines.reflex.components.flink.streaming.FlinkStreamingComponentFactory
 import com.parallelmachines.reflex.components.spark.batch.SparkBatchComponentFactory
-import com.parallelmachines.reflex.factory.{ReflexComponentFactory, TensorflowComponentFactory}
+import com.parallelmachines.reflex.factory.{PythonComponentFactory, ReflexComponentFactory}
```

Comment:
would it make sense to keep these naming consistent? Generic instead of Python?

Reply:
Yes, but that's a bigger project. We would need to look at how R is being run, etc.